### DISCUSSION TITLE
airflow audit log

### DISCUSSION THREAD

**Discussion by MicleIvle** [apache/airflow#28491] [Q&A]
Hello,
My airflow configured to integration with ACTIVE DIRECTORY (in airflow.cfg) and login works, but i can find any audit logs of login.
Has anyone experienced this? Need simple information in audit about ldap_username and time when login was.
Aairflow (version 2.3.3).

**Comment by potiuk** [comment]
I gues you need to track it in active directory then. Why do you look for it in Airflow?
